samples/src/main/java/AdvancedQuery.java

```diff
@@ -0,0 +1,83 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+import com.microsoft.azure.kusto.data.ClientImpl;
+import com.microsoft.azure.kusto.data.ClientRequestProperties;
+import com.microsoft.azure.kusto.data.HttpClientProperties;
+import com.microsoft.azure.kusto.data.KustoOperationResult;
+import com.microsoft.azure.kusto.data.KustoResultSetTable;
+import com.microsoft.azure.kusto.data.auth.ConnectionStringBuilder;
+
+import java.time.LocalDateTime;
+import java.util.ArrayList;
+import java.util.List;
+
+public class AdvancedQuery {
+    public static void main(String[] args) {
+
+        try {
+            ConnectionStringBuilder csb = ConnectionStringBuilder.createWithAadApplicationCredentials(
+                    System.getProperty("clusterPath"),
+                    System.getProperty("appId"),
+                    System.getProperty("appKey"),
+                    System.getProperty("appTenant"));
+
+            HttpClientProperties properties = HttpClientProperties.builder()
+                    .keepAlive(true)
+                    .maxKeepAliveTime(120)
+                    .maxConnectionsPerRoute(40)
+                    .maxConnectionsTotal(40)
+                    .build();
+
+            ClientImpl client = new ClientImpl(csb, properties);
+            String database = System.getProperty("dbName");
+            String newLine = System.getProperty("line.separator");
```

Comment:
why needed

Reply:
It's used to separate the lines of the query below for readability. Would you prefer to use a different character instead?

Reply:
just asked if there was a reason for the way you acquire it